src/panels/Menu/TopMenuBar/Menus/ViewMenu.tsx

```diff
@@ -62,15 +73,39 @@ export function ViewMenu() {
       </TopBarMenuWrapper>
 
       <Menu.Item leftSection={<SaveIcon />}>Load/Save Layout</Menu.Item>
-      <Menu.Divider />
-      <Menu.Label>
-        <Group>
-          <VisibilityIcon /> User Visibility
-        </Group>
-      </Menu.Label>
-      <Container>
-        <Property uri={'OpenSpaceEngine.PropertyVisibility'} />
-      </Container>
+      <TopBarMenuWrapper
+        targetTitle={
+          <Menu.Item
+            leftSection={<VisibilityIcon />}
+            rightSection={<ChevronRightIcon size={IconSize.sm} />}
+          >
+            User Visibility
+          </Menu.Item>
+        }
+        position={'right-start'}
+        withinPortal={false}
+        closeOnItemClick={false}
+      >
+        <Menu.Label>
+          <Group justify={'space-between'}>
+            Set visibility level
+            <InfoBox
+              text={`Controls what settings will be exposed in the interface. Increase the
+                level to reveal more advanced settings.`}
+            />
+          </Group>
+        </Menu.Label>
+        <Radio.Group
+          value={propertyVisibility?.toString()}
+          onChange={(newValue) => setPropertyVisibility(parseInt(newValue))}
+        >
```

Comment:
The reason why the up/down keys do not work is because the event "bubbles" and gets handled by the top level. if you add `onKeyDown={(event) => event.stopPropagation()}`, it works. :) 

The accessibility guidelines for radio inputs requires up/down to work as well as left/right: https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio-rating/

```suggestion
        <Radio.Group
          value={propertyVisibility?.toString()}
          onChange={(newValue) => setPropertyVisibility(parseInt(newValue))}
          onKeyDown={(event) => event.stopPropagation()}
        >
```

Reply:
Guessing we had the same issue with the ComboBox then, so if we liked it more, we could bring it back with the same fix. 

Personally, I think it more clearly shows the current state. But, the radio buttons more clearly show the states that are available.